static/third/bootstrap-typeahead/typeahead.js

```diff
@@ -93,6 +93,11 @@
  *   only this diff, and not the entire text, is highlighted when undoing,
  *   as would be ideal.
  *
+ * 9. Rerender on window resize:
+ * 
+ *   We add a new event handler on `window`, resizeHandler`, that calls the
+ *   lookup function again if the typeahead is already open.
+ * 
```

Comment:
As suggested in another comment, replace `lookup` with `show`, and also add here that calling this function calculates and sets the appropriate position, ensuring that the menu is where it should be on resizing. In general, try to explain why you did something apart from just saying what you did in a comment.